packages/pigeon/lib/dart_generator.dart

```diff
@@ -134,16 +135,16 @@ void _writeFlutterApi(
                   : func.returnType == 'void'
                       ? 'return;'
                       : 'return null;';
-              indent.write('if (message == null) ');
-              indent.scoped('{', '}', () {
-                indent.writeln(emptyReturnStatement);
-              });
               String call;
               if (argType == 'void') {
+                indent.writeln('// ignore message');
```

Comment:
Is `message` always `null` when `argType` is `void`?

Reply:
Do you mean in pigeon-generated code? Or?

Reply:
yup. pigeon-generated code

Reply:
it looks like `_writeHostApi` always sends null if the argType is void, but of course people can always bypass pigeon and send whatever.